server/routes/visit.test.ts

```diff
@@ -332,6 +248,37 @@ describe('GET /visit/:reference', () => {
       })
   })
 
+  // Temporarily hiding any locations other than Hewell pending more work on transfer/release (see VB-907, VB-952)
```

Comment:
This new test is actually the only real change to this file; pretty much all the above is just me refactoring slightly to remove duplicate test data.